Testscripts/Windows/VERIFY-INFINIBAND-MultiVM.ps1

```diff
@@ -1,424 +1,423 @@
+				-password $password "/root/SetupRDMA.sh" -RunInBackground
+			Wait-Time -seconds 2
+		}
 
-        $timeout = New-Timespan -Minutes 120
-        $sw = [diagnostics.stopwatch]::StartNew()
-        while ($sw.elapsed -lt $timeout){
-            $vmCount = $AllVMData.Count
-            foreach ($VMData in $AllVMData) {
-                Wait-Time -seconds 15
-                $state = Run-LinuxCmd -ip $VMData.PublicIP -port $VMData.SSHPort -username $user -password $password "cat /root/state.txt" -runAsSudo
-                if ($state -eq "TestCompleted") {
-                    $setupRDMACompleted = Run-LinuxCmd -ip $VMData.PublicIP -port $VMData.SSHPort -username $user -password $password `
-                        "cat /root/constants.sh | grep setup_completed=0" -runAsSudo
-                    if ($setupRDMACompleted -ne "setup_completed=0") {
-                        Throw "SetupRDMA.sh run finished on $($VMData.RoleName) but setup was not successful!"
-                    }
-                    Write-LogInfo "SetupRDMA.sh finished on $($VMData.RoleName)"
-                    $vmCount--
-                }
-            }
-            if ($vmCount -eq 0){
-                break
-            }
-            Write-LogInfo "SetupRDMA.sh is still running on $vmCount VM(s)!"
-        }
-        if ($vmCount -eq 0){
-            Write-LogInfo "SetupRDMA.sh is done"
-        } else {
-            Throw "SetupRDMA.sh didn't finish at least on one VM!"
-        }
+		$timeout = New-Timespan -Minutes 120
+		$sw = [diagnostics.stopwatch]::StartNew()
+		while ($sw.elapsed -lt $timeout){
+			$vmCount = $AllVMData.Count
+			foreach ($VMData in $AllVMData) {
+				Wait-Time -seconds 15
+				$state = Run-LinuxCmd -ip $VMData.PublicIP -port $VMData.SSHPort -username $user -password $password "cat /root/state.txt" -runAsSudo
+				if ($state -eq "TestCompleted") {
+					$setupRDMACompleted = Run-LinuxCmd -ip $VMData.PublicIP -port $VMData.SSHPort -username $user -password $password `
+						"cat /root/constants.sh | grep setup_completed=0" -runAsSudo
+					if ($setupRDMACompleted -ne "setup_completed=0") {
+						Throw "SetupRDMA.sh run finished on $($VMData.RoleName) but setup was not successful!"
+					}
+					Write-LogInfo "SetupRDMA.sh finished on $($VMData.RoleName)"
+					$vmCount--
+				}
+			}
+			if ($vmCount -eq 0){
+				break
+			}
+			Write-LogInfo "SetupRDMA.sh is still running on $vmCount VM(s)!"
+		}
+		if ($vmCount -eq 0){
+			Write-LogInfo "SetupRDMA.sh is done"
+		} else {
+			Throw "SetupRDMA.sh didn't finish at least on one VM!"
+		}
 
-        # Reboot VM to apply RDMA changes
-        Write-LogInfo "Rebooting All VMs!"
-        $TestProvider.RestartAllDeployments($AllVMData)
+		# Reboot VM to apply RDMA changes
+		Write-LogInfo "Rebooting All VMs!"
+		$TestProvider.RestartAllDeployments($AllVMData)
 
-        # In some cases, IB will not be initialized after reboot
-        Resolve-UninitializedIB
```

Comment:
this function was introduced for SLES testing - on 32 VMs test cases some of the VMs didn't initialize the IB after the first reboot. But after a secondary reboot, all was good. We need to run this function at least for SLES use case

Reply:
Will make sure of SLES15 and SLES 12 SP4 testing.

Reply:
> this function was introduced for SLES testing - on 32 VMs test cases some of the VMs didn't initialize the IB after the first reboot. But after a secondary reboot, all was good. We need to run this function at least for SLES use case

@adriansuhov , Do we have an active bug for this issue in TFS?

Reply:
@iamshital , @adriansuhov is working with internal teams for further investigation. Will find more data soon.

Reply:
Fixed in 33d1e401d9b470b70444c43d69ac962292bad790